doc/rippled-example.cfg

```diff
@@ -609,6 +609,15 @@
 #
 #
 #
+# [validator_key_revocation]
+#
+#   If a validator's keys have been compromised, a revocation must be generated
```

Comment:
I think I prefer only supporting a single revocation in the config. The validator-keys tool currently uses the same wording for generated tokens and revocations:
```
Update rippled.cfg file with these values and restart rippled:
```
The expectation for the user is to either add the field (if it's not already present) or replace the field's current contents.
I'll try to make the documentation more explicit both here and in the tool.